package.json

```diff
@@ -88,15 +88,12 @@
     "compression": "^1.7.4",
     "css-scss": "^0.1.1",
     "decomment": "^0.9.1",
-    "del": "^3.0.0",
     "dialog-polyfill": "^0.5.6",
     "dotenv": "^8.2.0",
     "ghp": "^1.3.1",
     "git-branch": "^2.0.1",
     "glob": "^7.1.6",
-    "global": "^4.4.0",
```

Comment:
@zhawkins There were several packages no longer in use so I removed them and tested that the builds continued to work.